README.md

```diff
@@ -53,6 +54,17 @@ By supplying `--all` to c8, all files in directories specified with `--src` (def
 and `--exclude` flag checks, will be loaded into the report. If any of those files remain uncovered they will be factored
 into the report with a default of 0% coverage.
 
+## Source-Map support for pre-instrumented codebases
+
+If you opt to pre-instrument your source-code (rather than using a just-in-time
+transpiler like [`@babel/register`]) c8 supports both inline source-maps and
+`.map` files.
+
+_Important: If you are using c8 with a project that pre-instruments its code,
```

Comment:
It would be useful to give an example of when we would want to set `--exclude-after-remap` to `true`, vs, when we would want to leave it `false`, rather than this blurb.